### PR TITLE
implement pom for tshirt test

### PR DESCRIPTION
**Pull request by alvaroagr**

### CODE REVIEW

File: src/page/order-summary.page.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+export class OrderSummaryPage {
+  private confirmationText: ElementFinder;
+
+  constructor() {
+    this.confirmationText = $('#center_column > div > p > strong');
+  }
+
+  public async confirmOrder(): Promise<void> {
+    await expect(this.confirmationText.getText())
```
Comment (holgiosalos) [inline src/page/order-summary.page.ts:11]: The idea with POM is just encapsulating business actions, but, verifications or expects should go on the test class. Please make this function to return a string and make the expect on the test class.

File: test/buy-tshirt.spec.ts [hunk 2]
```diff
@@ -1,41 +1,54 @@
...
+    await paymentStepPage.goToBankPaymentPage();
     await(browser.sleep(5000));
-    await $('#cart_navigation > button > span').click();
+
+    await bankPaymentPage.confirmOrder();
     await(browser.sleep(5000));
 
-    await expect($('#center_column > div > p > strong').getText())
-      .toBe('Your order on My Store is complete.');
+    await orderSummaryPage.confirmOrder();
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts:52]: expect should remain here

Comment (alvaroagr) [reply test/buy-tshirt.spec.ts]: it's been fixed.

File: test/buy-tshirt.spec.ts [hunk 3]
```diff
@@ -1,41 +1,55 @@
...
+    await paymentStepPage.goToBankPaymentPage();
     await(browser.sleep(5000));
-    await $('#cart_navigation > button > span').click();
+
+    await bankPaymentPage.confirmOrder();
     await(browser.sleep(5000));
 
-    await expect($('#center_column > div > p > strong').getText())
-      .toBe('Your order on My Store is complete.');
+    // await orderSummaryPage.getOrderText();
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts:52]: remove the comment. then you can squash and merge

**Pull request merged**